lib/sync/transforms/pull-request.js

```diff
@@ -25,40 +33,52 @@ function getAuthor(author) {
   }
 
   return {
-    avatar: author.avatarUrl,
+    avatar: author.avatar_url,
     name: author.login,
     url: author.url,
   };
 }
 
-module.exports = (payload, author) => {
-  // eslint-disable-next-line camelcase
-  const { pull_request: pullRequest, repository } = payload;
+/**
+ * @param {{pullRequest: import('@octokit/rest').Octokit.PullsListResponseItem, repository: import('../pull-request').RepositoryObject}} payload - The pull request details
+ * @param {import('@octokit/rest').Octokit.PullsListResponseItemUser} author - The pull request author
+ * @param {import('probot').GitHubAPI} github - The GitHub API Object
+ */
+module.exports = async (payload, author, github) => {
+  const { pullRequest, repository } = payload;
   const { issueKeys } = parseSmartCommit(pullRequest.title);
 
   if (!issueKeys) {
     return {};
   }
 
+  // Comment Count is required by the Jira API, but only do this second fetch if we have issue keys.
```

Comment:
Why? I dont get this sentence 🙈

Reply:
We must tell Jira the count of comments on the pull request. But there's no reason to make this second API call if there isn't a commit in the title. That's why it occurs after the `if (!issueKeys)`

Reply:
> if there isn't a commit

That makes sense

> Comment Count is required by the Jira API, but only do this second fetch if we have issue keys.

I'm just not sure how commits apply to this sentence

Reply:
if there isn't an **Issue Key** in the title. Wow that was confusing.